corehq/apps/cloudcare/static/cloudcare/js/formplayer/spec/menu_list_spec.js

```diff
@@ -28,8 +28,6 @@ hqDefine("cloudcare/js/formplayer/spec/menu_list_spec", function () {
                 user.displayOptions = {
                     singleAppMode: false,
                 };
-
-                hqImport("cloudcare/js/formplayer/apps/api").primeApps(user.restoreAs, new Backbone.Collection());
```

Comment:
Removing this one doesn't change test output. I'm guessing it used to be necessary but no longer is.